step-capstone/src/components/Flight.js

```diff
@@ -1,26 +1,25 @@
-import React from 'react'
-import { Grid, Typography } from '@material-ui/core'
-
+import React from 'react';
+import '../styles/TimeLine.css';
+ 
 /*
 Takes data in this form:
-        {
-          finalized: true,
-          type: "flight",
-          departureAirport: "BOS",
-          arrivalAirport: "SFO",
-          departureDate: "4:00pm EST",
-          arrivalDate: "7:00pm PST",
-          description: "Additional notes"
-        }
-
+       {
+         finalized: true,
+         type: "flight",
+         departureAirport: "BOS",
+         arrivalAirport: "SFO",
+         startDate: "4:00pm EST",
+         endDate: "7:00pm PST",
+         description: "Additional notes"
+       }
+ 
 */
-export default function Flight( data ) {
-    return(
-        <Grid item>
-            <Typography variant="h6" gutterBottom>Flight from { data.departureAirport } to { data.arrivalAirport }</Typography>
-            <Typography variant="subtitle2" gutterBottom>Departure: { data.departureDate.toString() }</Typography>
-            <Typography variant="subtitle2" gutterBottom>Arrival: { data.arrivalDate.toString() }</Typography>
-            <Typography variant="body2" gutterBottom>{ data.description }</Typography>
-        </Grid>
-    )
-}
\ No newline at end of file
+export default function Flight( props ) {
+   return(
+       <div className="event double flights" style={props.styleConfig}>
+           <p>{props.data.startDate.toLocaleString()} - {props.data.endDate.toLocaleString()}: Flight from { props.data.departureAirport } to { props.data.arrivalAirport }</p>
```

Comment:
I think this has potential to be better formatted: for example, having some sort of header (preferably depart/arrival airports), and under, the the corresponding times. It does take a bit more space but I think the one line of sparation could add a good amount to readability and UX. Also since the boxes are so small, chances are the text will be overflowed into the next line anyways.
Also, just for consistence, I think we should probably also stick with using Typography from Material UI for all our text (they do have a paragraph text option though I think we should bold or make bigger the airports since p tags are very small).

Reply:
Same with others (event, hotel)

Reply:
Good point.

Reply:
done